index.html

```diff
@@ -6,11 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
-    </head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
 <body>
-    <header>
-        <img src="" alt="">
+    <header id="toolbar">
+        <img src="images/ic_menu.svg" alt="Menu">
+        <a href=""></a> 
```

Comment:
Kindly wrap this inside a `nav` tag. This is for semantics reasons, and you are going to implement a menu using this button in future projects. :+1: :smiley: :smiley:

Reply:
Thanks for the sugestion, I'll changes